src/modules/fancyzones/UnitTests-FancyZonesEditor/DefaultLayoutsModelTests.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using FancyZonesEditor.Models;
+
+namespace UnitTestsFancyZonesEditor;
+
+[TestClass]
+public class DefaultLayoutsModelTests
+{
+    [TestMethod]
+    public void SettingFirstLayoutShouldBeLengthOne()
+    {
+        var defaultLayoutsModel = new DefaultLayoutsModel();
+        GridLayoutModel gridLayoutModel = new GridLayoutModel();
+
+        defaultLayoutsModel.Set(gridLayoutModel, MonitorConfigurationType.Horizontal);
+
+        var actualCount = defaultLayoutsModel.Layouts.Count;
+        Assert.AreEqual(1, actualCount);
+    }
```

Comment:
I don't clearly understand the purpose of this test.

Reply:
This test was to make sure that when you set one of the defaults that layouts count did increase. Showing that something was added to the layouts. This was a little more of a stepping stone test to make sure we understood the way the models were being used. Can be removed if deemed appropriate